tests/unit/nupic/frameworks/opf/opf_metrics_test.py

```diff
@@ -758,6 +758,34 @@ def testCustomErrorMetric(self):
       pass
 
 
+  def testMultiMetric(self):
+    ms1 = MetricSpec(field='a', metric='trivial',  inferenceElement='prediction', params={'errorMetric': 'aae', 'window': 1000, 'steps': 1})
+    ms2 = MetricSpec(metric='trivial', inferenceElement='prediction', field='a', params={'window': 10, 'steps': 1, 'errorMetric': 'rmse'})
+    metric1000 = getModule(ms1)
+    metric10 = getModule(ms2)
+    # create multi metric
+    multi = MetricMulti(weights=[0.2, 0.8], metrics=[metric10, metric1000], id='multi1')
+    multi.verbosity = 1
+    print multi 
+    # create reference metrics (must be diff from metrics above used in MultiMetric, as they keep history)
+    metric1000ref = getModule(ms1)
+    metric10ref = getModule(ms2)
+
+    
+    gt = [(i/4+1) for i in range(500, 1000)]
```

Comment:
it's not. but actually the ranges do not matter much, I just copied them from other test.